.buildkite/commands/build.sh

```diff
@@ -1,21 +1,5 @@
 #!/bin/bash -u
 
-echo "--- Set up SPM"
-PRIVATE_REPO_FETCH_KEY_NAME=private_repos_key
-add_ssh_key_to_agent "$PRIVATE_REPOS_BOT_KEY" $PRIVATE_REPO_FETCH_KEY_NAME
```

Comment:
We should make sure that we also:
 - Revoke/rotate the SSH key that was used in `$PRIVATE_REPOS_BOT_KEY`
 - Remove any of those unused key from `~/.mobile-secrets/CI/secrets/pocket-casts/ios/` too (both private key files in that dir that might now be unused, _and_ the `PRIVATE_REPOS_BOT_KEY` value in the `env` file of that repo)


To clean up after ourselves and not risk still having those accessible in non-trusted agents anymore (because even if we don't use them, if they are still in mobile-secrets and installed on agents, a malicious actor could still discover and use them otherwise)